src/components/_button.scss

```diff
@@ -8,10 +8,10 @@
   $type: 'primary',
   $size: 'medium',
   $layout: null,
-  $color: null,
-  $bgcolor: null,
-  $icon-fill: null,
-  $icon-stroke: null
+  $background-color: $c-gray-dark,
+  $color: $c-white,
+  $icon-fill: transparent,
+  $icon-stroke: $c-white
```

Comment:
Since this variables are only for custom buttons, should we use `$bgc-custom-button`?
maybe it will be more easy to understand